and-bible/app/src/main/java/net/bible/android/view/activity/page/MenuCommandHandler.java

```diff
@@ -89,7 +89,7 @@ public boolean handleMenuRequest(MenuItem menuItem) {
 		        	handlerIntent = new Intent(callingActivity, SettingsActivity.class);
 		        	// force the bible view to be refreshed after returning from settings screen because notes, verses, etc. may be switched on or off
 		        	mPrevLocalePref = CommonUtils.getLocalePref();
-		        	requestCode = IntentHelper.REFRESH_DISPLAY_ON_FINISH;
+		        	requestCode = IntentHelper.RESTART_REQUIRED;

```

Comment:
I don't think this is correct now.  REFRESH_DISPLAY_ON_FINISH is used when settings may have changed.  If the locale setting is changed then a restart is also required.  I suspect this will not cause a refresh for non locale changes e.g. change theme or show footnotes.  At bestthe name is now wrong.